pkgs/applications/radio/urh/default.nix

```diff
@@ -33,7 +33,7 @@ python3Packages.buildPythonApplication rec {
     homepage = "https://github.com/jopohl/urh";
     description = "Universal Radio Hacker: investigate wireless protocols like a boss";
     license = licenses.gpl3;
-    platforms = platforms.linux;
+    platforms = platforms.unix;
```

Comment:
`unix` does not include Darwin and Linux only.

Change the commit description accordingly.